src/Objs/Product/ProductComponent.tsx

```diff
@@ -206,24 +212,60 @@ const CartActionButton = connect(function CartActionButton({
     )
   }
 
-  if (isInCart(product)) {
+  const quantity = quantityInCart(product)
+  if (quantity) {
     return (
-      <button
-        className="btn btn-sm btn-primary"
-        onClick={() => {
-          removeFromCart(product)
-          toast.info(cartRemovedMessage)
-        }}
-      >
-        <i className="bi bi-x-lg"></i>
-        {cartRemoveLabel}
-      </button>
+      <div className="row mt-0 mb-1 g-1">
+        <div className="col-auto">
+          <div className="input-group input-group-sm flex-nowrap">
+            <button
+              aria-label="-"
+              className="btn btn-primary"
+              disabled={quantity < 2}
+              onClick={() => subtractFromCart(product)}
+            >
+              <i className="bi bi-dash-lg px-0" />
+            </button>
+            <input
```

Comment:
At least on Brave there are additional up/down arrows. Please remove the grey up/down icons:
<img width="149" alt="Screenshot 2025-03-04 at 12 43 32" src="https://github.com/user-attachments/assets/2e837fce-53bd-43d7-bfe7-53acd408dabe" />